README.md

```diff
@@ -1,5 +1,4 @@
-DevCycle CLI
-=================
+# DevCycle CLI
```

Comment:
how come this generated differently?

Reply:
dont know but reverted the changes

Reply:
was a version mismatch, removed node_modules and rebuild and all is good now